main.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener("DOMContentLoaded", function () {
+	let menu = document.querySelector(".menu-mobile");
```

Comment:
Actually, if you do not reassign the value of your variables, it's recommended to use `const`. :) At some point of time you'll probably also use ESLint which will enforce such usage. :)

https://stackoverflow.com/questions/41086633/why-most-of-the-time-should-i-use-const-instead-of-let-in-javascript

Reply:
oh.. Thank You. I should have known.. 🙄